### DISCUSSION TITLE
Bug Report: week crashing

### DISCUSSION THREAD

**Discussion by Ibexlol** [KadeArchive/Kade-Engine#502] [Q&A]
I've added my new week but when i try to play it, it crashes what do i do?

**Comment by isakube** [comment]
(Discussion)

Can you provide the terminal output when it crashes? Listing any errors you receive when you play the week would help.

**Reply by Ibexlol** [reply]
When i go down to play the week it freezes and crashes and shows no error in the terminal output

**Reply by isakube** [reply]
double check all your assets I guess, and compare them to the guide on the website. If you're 100% sure everything was done correctly, try looking through [closed issues](https://github.com/KadeDev/Kade-Engine/issues?q=week+crash+is%3Aissue+is%3Aclosed+) i guess to see if any solutions pop up. Another thing to consider is when it crashes. Does it crash immediately when you press Enter or does it crash when you hover over the week? because that can kinda help you pinpoint the problem a bit.
